### DISCUSSION TITLE
(DOMAIN) is already in use (Help please!!!)

### DISCUSSION THREAD

**Discussion by im2crispy** [NginxProxyManager/nginx-proxy-manager#2140] [Q&A]
Hello, appreciate your help!! ❤️

I am learning about NPM, and I started a VPS (Ubuntu EC2) and installed NPM Docker Container, configured the SSL to my domain, and at this time, all is fine. BUT...

I got an error on another thing and needed to delete this VPS and launch new ubuntu, did all the setup again, and when I tried to configure the SSL again, I just got this message: "SyntaxError: Unexpected token < in JSON at position 0"

And after "(mydomain) is already in use". I imagine that's because I already configured this same domain in the old server, but I don't have any more access, and now I can't set up my SSL through NPM. 

Can I revoke the old SSL or something like that? I imagine the SSL is linked to my domain (which I have access, of course) and exists any way to get a new SSL with NPM in this new VPS. 

Can you help me? Appreciate a lot! ❤️

**Comment by jc21** [comment]
If you have access to the docker logs for npm, it would be more helpful. You can delete SSL certs from within NPM itself and also manually remove them from certbot inside the docker container.

However it sounds like you've created an entirely new instance of npm and it's data, so not sure how this is happening for you at all.

**Reply by im2crispy** [reply]
Yeah, I created an entirely new instance of npm. I couldn't use another domain because it's a business, and this is the official domain. I am looking for a way to create an SSL on this domain, probably I need to revoke the old one, I just couldn't find much useful info about this

**Reply by jc21** [reply]
If you mean revoke it with let's encrypt, I'm not sure you can and I'm not sure it's the problem. I have been able to create many certs for the same domain using let's encrypt before. Maybe there's some new limitations for that. Maybe it's something with using HTTP or DNS validation...

**Reply by im2crispy** [reply]
That's the problem:

![image](https://user-images.githubusercontent.com/106400790/177663748-e8fd4b93-5256-4230-aa96-ae3b3400d019.png)

As you can see in the background, I don't have any test.kaulaeducacao.com.br SSL, but the NPM says to me that I already have. Do you have any suggestions?

**Reply by jc21** [reply]
This isn't a SSL problem, it's a host problem. You must already have `test.kaulaeducacao.com.br` defined as a host in either Proxy, Redirection or 404 section.

**Reply by im2crispy** [reply]
Yeah, you're right! As well [space.kaulaeducacao.com.br](url) isn't proxied. In fact I already added to Cloudflare, but the proxy is off:

![image](https://user-images.githubusercontent.com/106400790/177665167-ae165c88-37c4-4003-b69e-3a601dc71b85.png)

Because when I turn on the Cloudflare proxy on this subdomain I got this:

![image](https://user-images.githubusercontent.com/106400790/177665259-c9a22046-5c31-4967-a817-ddc768ead350.png)

But yeah, probably this is the problem. Now I just need to solve this Cloudflare problem hahaha! Thank you so much, and also, If you have any other suggestions, I will appreciate so much! 

As I said, I am very new on this, and trying to learn and help some business also.

**Reply by im2crispy** [reply]
I turned off Cloudflare just for testing purposes. I am trying to get an SSL to `space.kaulaeducacao.com.br` with NPM. I already have this subdomain on my DNS with an A registry:

![image](https://user-images.githubusercontent.com/106400790/177688618-5ac7037d-fbce-4178-9751-3e5c0b11944b.png)

But when I open NPM in `http://space.kaulaeducacao.com.br:81/nginx/proxy` and try to generate an SSL, I get the exact same error `"SyntaxError: Unexpected token < in JSON at position 0"` 

![image](https://user-images.githubusercontent.com/106400790/177688870-7ea23792-9e14-42ca-b70c-19824851aa82.png)

And the status of SSL is "Unknow". I really don't know what's happening

**Reply by im2crispy** [reply]
Another test I did was try to add the SSL to another domain (to see if it is problem of mine) but apparently, I got the same error with a random domain like `dasdadasfa.com`. Looks like it is a problem on my NPM, I will try to reinstall

**Reply by im2crispy** [reply]
Nop, I reinstalled NPM through portainer with this docker-compose: [https://nginxproxymanager.com/setup/#using-mysql-mariadb-database=](url) but I got the exact same error `SyntaxError: Unexpected token < in JSON at position 0`. What's happening?

**Reply by im2crispy** [reply]
Heyy! I woke up today and after some hours, the SSL changed the state from "Unknow" to "Online", this is great!!

![image](https://user-images.githubusercontent.com/106400790/177793294-1cf70938-8191-45be-8dd1-e54404b3e515.png)

The bad part is, still didn't work :( I tried to access [https;/space.kaulaeducacao.com.br](url) and didn't work, only http works.

Then, I tried to create a new SSL on port 81, to make sure the old problem is gone, but when I created, I got the same error: `SyntaxError: Unexpected token < in JSON at position 0` probably tomorrow the SSL will appear "Online", but don't work

I can't understand this problem, already tried to make full reinstall of NPM, but still the same

**Reply by im2crispy** [reply]
Hello!! Anyone? Still have this problem :(

**Reply by TmmTohru** [reply]
I just discover this problem and due to the same circumstances as OP.

**Reply by nibazshab** [reply]
I have the same problem.

**Reply by DanielBaumert** [reply]
Same

**Reply by emden09g** [reply]
same circumstances. Old server was broken, so I did setup a new server with docker-compse.yaml I stil had. As I had no backupfiles of nginx proxy manager I did set it up entirely new. when trying to ad a new host (I changed the IP on duckdns.org first to the Ip of the new machine) I first got internal error when trying to cick o save after that I did get *.<my-domain name> is already in use.
